.github/ISSUE_TEMPLATE/alpha_feature.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Alpha Feature Requirements
+about: Requirements for promoting a feature to alpha
+---
+
+This page lists the requirements for promoting a feature to alpha. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] Requires approved RFC in order to merge. 
+
+**Config**
+
+- [ ] Requires explicit user action to enable (e.g. a config field, config resource, or installation action). 
+
+**Docs**
+
+- [ ] Reference docs are published to istio.io. 
+- [ ] Basic feature docs are published on istio.io describing what the feature does, how to use it, and any caveats. 
+- [ ] Whenever possible, documentation has automated istio.io tests written. 
+- [ ] A reference to the design doc/issue is published on istio.io. 
+- [ ] Release notes entries added as appropriate
+- [ ] Upgrade notes entries added as appropriate
+
+**Tests**
+
+- [ ] Test coverage is >= 70%
+- [ ] Integration tests cover core use cases with the feature enabled. 
```

Comment:
Can this be manual or does it have to be automated for alpha?

Reply:
My thought is these should probably be automated (anything we have automated means we don't have to hunt down users to manually test it before each release). Do you have an opinion?